From 8cafdf9f33432a6eedd40efe2adede6f7d8133f2 Mon Sep 17 00:00:00 2001
From: Marc Zyngier <marc.zyngier@arm.com>
Date: Thu, 30 Jun 2016 18:40:38 +0100
Subject: [PATCH 297/447] arm64: Add ARM64_HYP_OFFSET_LOW capability

Git-commit: 853c3b21ff35816a2ae351fd7c2adb101c1f4503
Patch-mainline: v4.8-rc1
References: fate#320512

As we need to indicate to the rest of the kernel which region of
the HYP VA space is safe to use, add a capability that will
indicate that KVM should use the [VA_BITS-2:0] range.

Signed-off-by: Marc Zyngier <marc.zyngier@arm.com>
Signed-off-by: Christoffer Dall <christoffer.dall@linaro.org>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 arch/arm64/include/asm/cpufeature.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/include/asm/cpufeature.h b/arch/arm64/include/asm/cpufeature.h
index 09de1f0..82b1da2 100644
--- a/arch/arm64/include/asm/cpufeature.h
+++ b/arch/arm64/include/asm/cpufeature.h
@@ -37,8 +37,9 @@
 #define ARM64_ALT_PAN_NOT_UAO			12
 #define ARM64_HAS_32BIT_EL0			13
 #define ARM64_HAS_VIRT_HOST_EXTN		14
+#define ARM64_HYP_OFFSET_LOW			15
 
-#define ARM64_NCAPS				15
+#define ARM64_NCAPS				16
 
 #ifndef __ASSEMBLY__
 
-- 
2.11.0

